frontend/src/App.tsx

```diff
@@ -16,31 +15,22 @@ function App() {
   return (
     <>
       <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
+        <a href='https://docs.squid.cloud' target='_blank'>
+          <img src={squidLogo} className='logo' alt='Squid Cloud logo' />
         </a>
       </div>
       <h1>Vite + React</h1>
-      <div className="card">
+      <div className='card'>
```

Comment:
I think we might want to install `prettier` in all of these repos (`npm install -D prettier`) can configure it in your IDE to run on save. Just will ensure that everything is consistent (prettier will make this double quotes).